client/src/core/services/parse-map.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ParseMapService {
+    terrain: any = {
+        name: 'Terrain 1',
+        height: 30,
+        width: 50,
+        obstacles: [
+        ],
+        trees: [
+        ],
+    };
+
+  constructor() { }
+getMap() {
+    return this.terrain;
+}
+
+parseFile(file: string) {
+    let height : number = 1;
+    let width : number = 1;
+    let coordy : number = 0;
+    let coordx : number = 0;
```

Comment:
Codacy found an issue: ['coordx' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5943029)